README.md

```diff
@@ -230,7 +290,7 @@ sudo apt-get install -y nfs-common
 | version           | Config file version number                                               | 1          |
 | storageDriverName | `ontap-nas`, `ontap-san`, `eseries-iscsi`, or `solidfire-san`            | ontap-nas  |
 | debug             | Turn debugging output on or off                                          | false      |
-| storagePrefix     | Optional prefix for volume names.  Default: "netappdvp_"                 | netappdvp_ |
+| storagePrefix     | Optional but recommended prefix for docker volume names.  Default: "netappdvp_"                 | netappdvp_ |
```

Comment:
While this renders fine in a rich text viewer, it would be best to update the column widths to make it line up properly in plain text as well